content/integrations/aws-secrets-manager.md

```diff
@@ -227,3 +227,135 @@ You need an [AWS access key and AWS secret key pair](https://docs.aws.amazon.com
+      kmskey: ""     # The AWS-KMS key ID used to en/decrypt secrets at the SecretsManager. By default (if not set) the default AWS-KMS key will be used.
+      credentials:   # The AWS credentials for accessing secrets at the AWS SecretsManager.
+        accesskey: "${AWS_ACCESS_KEY}"  # Your AWS Access Key
+        secretkey: "${AWS_SECRET_KEY}"  # Your AWS Secret Key
+        token: ""      # Your AWS session token (usually optional)
+```
+
+{{< /tab >}}
+
+{{< tab "Reference" >}}
+
+For complete documentation, see the [configuration page]({{< relref "/tutorials/configuration.md" >}}).
+
+| <div style="width:150px"> Key  </div>                        | Description                    |
+|-----------------------------|--------------------------------|
+| `address`                     | The network address and port the KES server listens to on startup. Defaults to port `7373` on all host network interfaces. |
+| `root`                        | The identity for the KES superuser (`root`) identity. Clients connecting with a TLS certificate whose hash (`kes identity of client.cert`) matches this value have access to all KES API operations. Specify `disabled` to remove the root identity and rely only on the `policy` configuration for controlling identity and access management to KES. |
+| `tls`                         | The TLS private key and certificate used by KES for establishing TLS-secured communications. Specify the full path for both the private `.key` and public `.cert` to the `key` and `cert` fields, respectively. |
+| `policy`                      | Specify one or more [policies]({{< relref "/tutorials/configuration.md#policy-configuration" >}}) to control access to the KES server. MinIO SSE requires access to the following KES cryptographic APIs: <br><br> `/v1/key/create/*` <br> `/v1/key/generate/*` <br> `/v1/key/decrypt/*` <br><br> Specifying additional keys does not expand MinIO SSE functionality and may violate security best practices around providing unnecessary client access to cryptographic key operations. <br><br> You can restrict the range of key names MinIO can create as part of performing SSE by specifying a prefix before the `*.` For example, `minio-sse-*` only grants access to `create`, `generate`, or `decrypt` keys using the `minio-sse-` prefix. <br><br>KES uses mTLS to authorize connecting clients by comparing the hash of the TLS certificate against the `identities` of each configured policy. Use the `kes identity of` command to compute the identity of the MinIO mTLS certificate and add it to the `policy.<NAME>.identities` array to associate MinIO to the `<NAME>` policy. |
+| `keys`                        | Specify an array of keys which *must* exist on the root KMS for KES to successfully start. KES attempts to create the keys if they do not exist and exits with an error if it fails to create any key. KES does not accept any client requests until it completes validation of all specified keys.|
```

Comment:
Would `fails to create any key` mean `fails to create one or more required keys`?

Reply:
one or more. FIxed throughout.